### ISSUE TITLE
Cleanup empty paths

### ISSUE THREAD

**Issue by khaledhosny** [googlefonts/picosvg#252]
I have a punch of SVGs that contain empty sub-paths (move followed immediately by close, e.g. `M465,550 Z`) and I’m wondering if picosvg should remove them since they are essentially a no-op? (if one is wondering how I ended up with these, they are anchors and the code I used to generate the SVGs helpfully tried to preserve them by creating such empty paths).

**Comment by anthrotype** [comment]
Well, picosvg does try to [remove_unpainted_shapes](https://github.com/googlefonts/picosvg/blob/3ed618a4c85a335ecc61973a45e2adb233c61083/src/picosvg/svg.py#L889) (i.e. ones whose area is 0). If it turns out it doesn't, then I'd consider it a bug.

**Comment by anthrotype** [comment]
@khaledhosny please post an example of such empty sub-paths that fail to be removed. If I try to process an svg containing your move-only path example, I get the expected result:

Input:
```xml
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1000">
  <path d="M465,550 Z"/>
</svg>
```

Result:
```xml
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1000">
  <defs/>
</svg>
```

**Comment by khaledhosny** [comment]
Sorry, I didn’t realized picosvg was doping this. In my case they were inside larger path definitions, e.g.:
```xml
<?xml version="1.0" standalone="no"?>
<svg   xmlns="http://www.w3.org/2000/svg"  viewBox="0 0 350.0 1500.0"  width="350.0"  height="1500.0">
 <g transform="matrix(1 0 0 -1 0 1100.0)">
  <path d="M-20.0 0.0 H370.0V100.0H-20.0V0.0ZM85.0 550.0ZM85.0 550.0ZM168.0 46.0 248.0 121.0 25.0 364.0 -41.0 273.0 168.0 46.0ZM155.0 50.0H255.0V400.0L155.0 378.0V50.0Z" />
 </g>
</svg>
```

Results in:
```xml
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 350.0 1500.0" width="350.0" height="1500.0">
  <defs/>
  <path fill="#404040" d="M-20,1100 L370,1100 L370,1000 L-20,1000 L-20,1100 Z M85,550 Z M85,550 Z M168,1054 L248,979 L25,736 L-41,827 L168,1054 Z M155,1050 L255,1050 L255,700 L155,722 L155,1050 Z"/>
</svg>
```

See the `M85,550 Z M85,550 Z` commands.

**Comment by rsheeter** [comment]
Makes sense to drop these imo, good catch.

**Comment by anthrotype** [comment]
After #261 the foot emoji looks like this. Tempoarily reverting this PR fixes the issue, so something here is the cause of the issue. I'll keep investigating.

<img width="597" alt="Screenshot 2022-02-01 at 13 19 26" src="https://user-images.githubusercontent.com/6939968/151976383-245e6501-4162-408d-85b8-0549de6178d4.png">

**Comment by rsheeter** [comment]
I'm under the impression that was fixed by #264 (ty) so closing. Plz reopen if that's not the case.
